packages/app/script/generatePreload.ts

```diff
@@ -0,0 +1,13 @@
+import { nativeBridgeRegistry } from '../src/nativeBridge/registry';
+import { close, openSync, writeSync } from 'fs-extra';
+
+const apiFile = nativeBridgeRegistry.generateAPIFile();
+const typeFile = nativeBridgeRegistry.generateAPITypeFile('./nativeBridge/modules/');
```

Comment:
would the nativeBridge/modules path ever change? if not it can be hidden as a constant inside the function

Reply:
probably not, I had a few things abstracted out when I xferred the impl over from my other project